src/main/java/lotto/LotteryCalculator.java

```diff
@@ -0,0 +1,13 @@
+package lotto;
+
+public class LotteryCalculator {
+    public static final int LOTTO_TICKET_UNIT_PRICE = 1000;
+
+    public static int calculateTicket(int money) {
+            return money/LOTTO_TICKET_UNIT_PRICE;
```

Comment:
```suggestion
            return money / LOTTO_TICKET_UNIT_PRICE;
```

컨벤션을 위반하셨습니다. IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
단축키 활용 방법 알려주셔서 감사합니다! ㅎㅎ